.github/workflows/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,41 @@
+# Code of Conduct
+
+## Request from out side
+
+We, the contributors and maintainers, commit to ensuring that everyone's participation in our project and community is free from harassment, regardless of age, body size, disability, ethnicity, gender identity and expression, experience level, education, socioeconomic status, nationality, personal appearance, race, religion, or sexual orientation. This is done in the spirit of creating a friendly and open environment.
+
+## Our Standards
+
+The following are some instances of actions that support the development of a pleasant environment:
+
+* Speaking in an open and accepting manner 
+* Respecting the opinions and experiences of others 
+* Taking constructive criticism in stride
+* Putting the good of the community first 
+* Demonstrating empathy for other community members
+
+Example of participant behavior that is undesirable include:
+
+* Public or private harassment 
+*The publication of another person's private information, such as a physical or electronic address, without that person's express consent 
+* The use of sexualized language or imagery and unwanted sexual attention or advances 
+* Trolling, offensive or derogatory remarks, and personal or political attacks 
+
+## Our Responsibilities
+
+Project maintainers are expected to take appropriate and equitable remedial action in response to any instances of undesirable behavior, as well as to clarify the standards of acceptable behavior.
+
+The right and obligation of project maintainers is to delete, modify, or reject comments, commits, code, wiki edits, issues, and other contributions that do not follow this code of conduct. They also have the authority to temporarily or permanently ban any contributor for any other actions they believe to be improper, threatening, offensive, or harmful.
+
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Reports of abusive, harassing, or otherwise inappropriate behavior can be sent to info@mifos.org, the project team's email address. After each complaint is examined and looked into, a response that is judged essential and fitting for the situation will be given. The project team has a duty to keep the identity of the incident reported discreet. Specific enforcement policies may have additional information posted separately.
```

Comment:
You can change this e-mail address to 'jeffrey@markr.ai' and 'vkehfdl1@gmail.com'